flatdata-go/flatdata/TODO.md

```diff
@@ -0,0 +1,8 @@
+Some todos for the golang implementation:
```

Comment:
I think we should create an epic containing this a task list after the PR is merged.

Reply:
Sure thing.